index.rst

```diff
@@ -2,28 +2,99 @@
 
 .. sectnum::
 
-.. Metadata such as the title, authors, and description are set in metadata.yaml
+Abstract
+========
 
-.. TODO: Delete the note below before merging new content to the main branch.
+The First-Look Analysis and Feedback Functionality (FAFF) working group, identified the need for a service that is capable of executing user-defined operations/computation following user-defined rules.
+This service has been referred  to as Catcher.
+In essence, the Catcher is not that different from the Watcher or other stream processing off-the-shelf technologies like Kapacitor.
+This technote explores the idea of using influx2.0 new stream processing capabilities to as an alternative implementation for the Catcher.
 
-.. note::
+Introduction
+============
 
-   **This technote is a work-in-progress.**
+One of the finding of the First-Look Analysis and Feedback Functionality (FAFF) working group :sitcomtn:`025` :cite:`SITCOMTN-025`, was the need for a service to ...
 
-Abstract
-========
+   handle the low-level coordiation of identifying when a specific condition is met, then launching and monitoring an analysis process.
+
+The original proposal from the FAFF working group was to develop a Commandable SAL Component (CSC), similar to the Watcher, that would provide be capable of monitoring data from the system and trigger certain actions (e.g. perform some computation and/or generate reports) based on some pre-defined conditions.
+
+Following additional discussions, we have decided to experiment with an alternative approach, replacing the CSC with an off-the-shelf solution provided by the Engineering Facility Database (EFD).
+
+As shown in :sqr:`034` :cite:`SQR-034`, the EFD has of an `influxDB <https://www.influxdata.com/>`__ time series database, that stores the data produced by the system.
+The release of influx2.0, brings a number of important updates, amongst which two are relevant for stream processing; `flux`_ and `influxDB task`_.
+
+.. _flux: https://docs.influxdata.com/flux/v0.x/
+.. _influxDB task: https://docs.influxdata.com/influxdb/cloud/process-data/get-started/
+
```

Comment:
I would reference the OSS docs whenever possible, there are some differences between the Cloud and the OSS versions. For InfluxDB Tasks: https://docs.influxdata.com/influxdb/v2.2/process-data/get-started/